fastlane/Fastfile

```diff
@@ -587,83 +719,102 @@ def send_message_to_mattermost(options)
   end
 end
 
-def submit_to_store
-  apk_path = lane_context[SharedValues::GRADLE_APK_OUTPUT_PATH]
-  ipa_path = lane_context[SharedValues::IPA_OUTPUT_PATH]
-
+def submit_to_testflight(ipa_path)
   app_name =  ENV['APP_NAME'] || 'Mattermost Beta'
   app_name_sub = app_name.gsub(" ", "_")
 
-  s3_region = ENV['AWS_REGION']
-  s3_bucket = ENV['AWS_BUCKET_NAME']
-
-  # Submit to Google Play if required
-  if !apk_path.nil? && ENV['SUBMIT_ANDROID_TO_GOOGLE_PLAY'] == 'true'
-    UI.success("apk file #{apk_path}")
-
-    supply(
-        track: ENV['SUPPLY_TRACK'] || 'alpha',
-        apk: apk_path
+  if(File.file?(ipa_path))
+    UI.success("ipa file #{ipa_path}")
+    upload_to_tesflight(
+        ipa: ipa_path
     )
+  else
+    UI.user_error! "ipa file does not exist #{ipa_path}"
+    return
+  end
 
-    version_number = android_get_version_name(gradle_file: './android/app/build.gradle')
-    build_number = android_get_version_code(gradle_file: './android/app/build.gradle')
-
-    s3_folder = "#{ENV['AWS_FOLDER_NAME']}/#{version_number}/#{build_number}"
-    public_link = "https://#{s3_bucket}/#{s3_folder}/#{app_name_sub}.apk"
+  version_number = get_version_number(xcodeproj: './ios/Mattermost.xcodeproj', target: 'Mattermost')
+  build_number = get_build_number(xcodeproj: './ios/Mattermost.xcodeproj')
+  s3_folder = "#{ENV['AWS_FOLDER_NAME']}/#{version_number}/#{build_number}"
+  public_link = "https://#{ENV['AWS_BUCKET_NAME']}/#{s3_folder}/#{app_name_sub}.ipa"
 
-    # Send a build message to Mattermost
-    pretext = '#### New Android released ready to be published'
-    msg = "Release has been cut and is on the Beta track ready to be published.\nDownload link: #{public_link}"
 
-    if ENV['BETA_BUILD'] == 'true'
-      pretext = '#### New Android beta published to Google Play'
-      msg = "Sign up as a beta tester [here](https://play.google.com/apps/testing/com.mattermost.rnbeta).\nDownload link: #{public_link}"
-    end
+  # Send a build message to Mattermost
+  pretext = '#### New iOS released ready to be published'
+  msg = "Release has been cut and is on TestFlight ready to be published.\nDownload link: #{public_link}"
```

Comment:
I was planning on doing something like this using https://circleci.com/orbs/registry/orb/tokenlab/mattermost-notify but in circle's config.
0/5 if it's better in the fastfile/makefiles or in circle

Reply:
This is not new really, is been there since the beginning of time, the reason it is here is because anyone building the apps outside of mattermost can use it without having to build the apps using circleCi

Reply:
I know, but I was thinking that the move to circle is so there is no need for people to generate relesases manually, so it might make sense to only have messages from circle.
Ofc Circle might fail, but then I might not really care about automated messaging, as we are probably dealing with bigger problems

Reply:
We cannot bound or force our community or customers to build the app with circleCI